UI/GameSettingsScreen.cpp

```diff
@@ -1137,6 +1143,74 @@ UI::EventReturn DeveloperToolsScreen::OnLoadLanguageIni(UI::EventParams &e) {
 	return UI::EVENT_DONE;
 }
 
+UI::EventReturn DeveloperToolsScreen::OnOpenTexturesIniFile(UI::EventParams &e) {
+	std::string gameID = g_paramSFO.GetValueString("DISC_ID");
+	std::string texturesDirectory = GetSysDirectory(DIRECTORY_TEXTURES) + gameID + "/";
+	bool enabled_ = !gameID.empty();
+	if (enabled_) {
+		if (!File::Exists(texturesDirectory)) {
+			File::CreateFullPath(texturesDirectory);
+		}
+		if (!File::Exists(texturesDirectory + "textures.ini")) {
+			FILE *f = File::OpenCFile(texturesDirectory + "textures.ini", "wb");
+			if (f) {
+				fwrite("\xEF\xBB\xBF", 0, 3, f);
+				fclose(f);
+			}
+		}
+		enabled_ = File::Exists(texturesDirectory + "textures.ini");
+	}
+	if (enabled_) {
+		std::string texturesIniFile;
+#if defined(_WIN32)
+		texturesIniFile = texturesDirectory + "textures.ini";
+		// Can't rely on a .txt file extension to auto-open in the right editor,
+		// so let's find notepad
+		wchar_t notepad_path[MAX_PATH + 1];
+		GetSystemDirectory(notepad_path, MAX_PATH);
+		wcscat(notepad_path, L"\\notepad.exe");
+
+		wchar_t ini_path[MAX_PATH + 1] = { 0 };
+		wcsncpy(ini_path, ConvertUTF8ToWString(texturesIniFile).c_str(), MAX_PATH);
+		// Flip any slashes...
+		for (size_t i = 0; i < wcslen(ini_path); i++) {
+			if (ini_path[i] == '/')
+				ini_path[i] = '\\';
+		}
+
+		// One for the space, one for the null.
+		wchar_t command_line[MAX_PATH * 2 + 1 + 1];
+		wsprintf(command_line, L"%s %s", notepad_path, ini_path);
+
+		STARTUPINFO si;
+		memset(&si, 0, sizeof(si));
+		si.cb = sizeof(si);
+		si.wShowWindow = SW_SHOW;
+		PROCESS_INFORMATION pi;
+		memset(&pi, 0, sizeof(pi));
+		UINT retval = CreateProcess(0, command_line, 0, 0, 0, 0, 0, 0, &si, &pi);
+		if (!retval) {
+			ERROR_LOG(COMMON, "Failed creating notepad process");
+		}
+		CloseHandle(pi.hThread);
+		CloseHandle(pi.hProcess);
```

Comment:
Should ideally factor this out into a common function for opening a text file, probably... I'm guessing this is repeated code from cheat inis.

-[Unknown]